src/prefect/client/orchestration.py

```diff
@@ -3012,11 +3012,16 @@ async def read_worker_metadata(self) -> Dict[str, Any]:
         return response.json()
 
     async def increment_concurrency_slots(
-        self, names: List[str], slots: int, mode: str
+        self, names: List[str], slots: int, mode: str, create_if_missing: Optional[bool]
     ) -> httpx.Response:
         return await self._client.post(
             "/v2/concurrency_limits/increment",
-            json={"names": names, "slots": slots, "mode": mode},
+            json={
+                "names": names,
+                "slots": slots,
+                "mode": mode,
+                "active": create_if_missing,
```

Comment:
it looks like you toggle between the names `create_if_missing` and `active` in various places, but I think the API here uses `create_if_missing`